docker/docker-compose.connect-popup-ci.yml

```diff
@@ -8,7 +8,7 @@ services:
     # network_mode: bridge # this makes docker reuse existing networks
 
   test-run:
-    image: mcr.microsoft.com/playwright:focal
+    image: mcr.microsoft.com/playwright:v1.41.2-jammy
```

Comment:
i see this is image is based on new ubuntu. definitely good thing to udpate it, but I am just curious, did we need it because of some problems with github?

Reply:
The reason I did it is because I was having some differences in the behavior of the test in GitHub, my machine and GitLab. So fixing it to a specific version removed one potential difference.

In general I think having a specific version is better, the only downside is that sometimes when playwright release a new version it forces you to update.